deploy/charts/approver-policy/README.md

```diff
@@ -1,6 +1,6 @@
 # cert-manager-approver-policy
 
-![Version: v0.4.2](https://img.shields.io/badge/Version-v0.4.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.4.0](https://img.shields.io/badge/AppVersion-v0.4.0-informational?style=flat-square)
+![Version: v0.0.0](https://img.shields.io/badge/Version-v0.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.0.0](https://img.shields.io/badge/AppVersion-v0.0.0-informational?style=flat-square)
```

Comment:
The helm-docs tool uses the static Chart.yaml values, when we generate the .tar.gz chart release the versions are set correctly. I think it is acceptable to keep these versions in the README as v0.0.0.